### ISSUE TITLE
vscode extension - inline bookmarks fails.

### ISSUE THREAD

**Issue by rholland** [eclipse-theia/theia#8160]
I am running Theia at Gitpod.  They directed me here.

visx extension here:
(https://marketplace.visualstudio.com/items?itemName=tintinweb.vscode-inline-bookmarks)

Fails with this:

![image](https://user-images.githubusercontent.com/3033195/87274557-dbc9ed80-c490-11ea-9661-25b43da15e07.png)

**Comment by vince-fugnitto** [comment]
@rholland thank you for the issue, do you have additional information that may be useful to help identify the problem?
- reproduction steps
- how did you install the extension

I installed the extension and it worked for me, it'd be good in the future to keep the issue template so we can better help:

<div align='center'>

![bookmark](https://user-images.githubusercontent.com/40359487/87423407-41f56400-c5a8-11ea-9db7-c11bc66bb80d.png)

</div>

---

@akosyakov I added the `needs more info` label since the extension worked for me, but it looks like my GitHub comment was gone.

**Comment by akosyakov** [comment]
@vince-fugnitto ah, thank you for confirming. I think Gitpod Theia needs update.

**Comment by rholland** [comment]
I just tried again and got the same message.  

Gitpod installs from here if you want to take a look at the gitpod files.
https://github.com/RivetCalc/rivetcalctemplate-001

The extension is here:
https://github.com/tintinweb/vscode-inline-bookmarks
I tried to install by drag and drop the vsix file.

Here is the console output:
![Untitled](https://user-images.githubusercontent.com/3033195/87823199-b853c980-c827-11ea-907a-17cf459f1872.png)
